docs/adr/0003-allowing-implicit-object-type.md

```diff
@@ -0,0 +1,44 @@
+# Allowing implicit object type
+
+## Status
+
+Accepted.
+
+## Context
+
+In real-world OpenAPI schema, we have discovered that people sometimes omit the `type: object` from their object definitions. We believe that this is valid Open API schema, if and only if the .
```

Comment:
This sentence grinds to a sudden and unexpected halt!

Reply:
Oops! Updated to remove the unnecessary dangling part.